OldTests/src/test/java/com/tle/webtests/test/contribute/MetadataMappingTest.java

```diff
@@ -71,8 +71,8 @@ public void imsMapping()
 		assertNode(x, "RepeatingReplace", "Vocabulary");
 		assertNode(x, "RepeatingReplace", "Travel");
 
-		assertNode(x, "HTML", "html description");
-		assertNoNode(x, "HTML", text);
+		assertNode(x, "HTML", text);
+		assertNoNode(x, "HTML", "html description");
```

Comment:
As per our discussions and investigations, it looks as though sometime in the past oEQ would set this empty element to `html description`. But now when we review the metadata/`<XML>` we see it is stored as an empty element (i.e. `<HTML/>`). So arguably this should be a check for that.

HOWEVER, this is a redundant check. We've already done an assertion for one value, and checking an inverse is not required. It would only be required if it was possibly to have multiple entries, however this is not a repeating control.

This line can be removed me thinks.